examples/lighting-app/nrfconnect/main/include/AppTask.h

```diff
@@ -46,6 +46,8 @@ class AppTask
 
     void DispatchEvent(AppEvent * event);
 
+    static void StartThread(void);
+
     static void FunctionTimerEventHandler(AppEvent * aEvent);
     static void FunctionHandler(AppEvent * aEvent);
     static void JoinerHandler(AppEvent * aEvent);
```

Comment:
This is misleading. Can you use e.g. `StartThreadHandler` name?

Reply:
Done